packages/bot-web-ui/src/stores/app-store.ts

```diff
@@ -49,17 +49,35 @@ export default class AppStore {
         link: localize('Switch to another account'),
     });
 
-    getErrorForEuClients = (is_logged_in = false) => {
+    getErrorForEuClients = (is_logged_in = false, country: string | undefined = undefined) => {
         return {
             text: ' ',
             title: is_logged_in
-                ? localize('Deriv Bot is not available for EU clients')
-                : localize('Deriv Bot is unavailable in the EU'),
+                ? localize(`Deriv Bot is not available for ${country || 'EU'} clients`)
+                : localize(`Deriv Bot is unavailable in ${country || 'the EU'}`),
             link: is_logged_in ? localize("Back to Trader's Hub") : '',
             route: routes.traders_hub,
         };
     };
 
+    throwErrorForExceptionCountries = (client_country: string) => {
+        const { client, common } = this.core;
+
+        const not_allowed_clients_country: { [key: string]: string } = {
+            au: 'Australian',
+            no: 'Norwegian',
+            jp: 'Japanese',
+            sg: 'Singaporean',
+        };
+
+        if (Object.prototype.hasOwnProperty.call(not_allowed_clients_country, client_country)) {
```

Comment:
use it this way instead of this one: 
`        if (not_allowed_clients_country.hasOwnProperty(client_country)) {...`
        Because of the error:` Do not access Object.prototype method 'hasOwnProperty' from target object`

Reply:
@maryia-matskevich-deriv wouldn't it be more simple this way?
```
    const not_allowed_clients_country: { [key: string]: string } = {
        au: 'Australian',
        no: 'Norwegian',
        jp: 'Japanese',
        sg: 'Singaporean',
    };

    const country_name = not_allowed_clients_country[client_country];

    if (country_name) {
        return showDigitalOptionsUnavailableError(
            common.showError,
            this.getErrorForEuClients(client.is_logged_in, country_name)
        );
    }
```

Reply:
If the property doesn't exist it would simply be undefined and the condition wouldn't pass. And we can reuse the `country_name` while calling `getErrorForEuClients`.

Reply:
good idea! thanks🤗 done